### PR TITLE
Fix link on styleguide page to homepage

### PR DESCRIPTION
**Pull request by RobLoach**
#11

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
     "browserify-shim": "^3.8.8",
     "holderjs": "^2.8.0",
     "jquery": "^2.1.4",
-    "kss-bootstrap": "~0.4.3",
+    "kss-bootstrap": "kalamuna/kss-bootstrap#3fb55ed3dd1c35dd118db6866cf2660f792a097d",
```
Comment (andrewmallis) [inline package.json]: @RobLoach what are the implications here?
Should we not roll a 0.4.4 version of kss-boostrap instead?

Comment (RobLoach) [inline package.json]: Rolling a 0.4.4 is much nicer solution.

**Comment by andrewmallis** [comment]
let's dance

File: package.json (hunk 1 above)
Comment (andrewmallis) [inline package.json]: let's dance.

**Pull request closed without merging**